core/solver/cg.cpp

```diff
@@ -148,25 +146,12 @@ void Cg<ValueType>::apply(const LinOp *alpha, const LinOp *b, const LinOp *beta,
 }
 
 
-template <typename ValueType>
-std::unique_ptr<LinOp> CgFactory<ValueType>::generate(
-    std::shared_ptr<const LinOp> base) const
-{
-    ASSERT_EQUAL_DIMENSIONS(base,
-                            size(base->get_num_cols(), base->get_num_rows()));
-    auto cg = std::unique_ptr<Cg<ValueType>>(Cg<ValueType>::create(
-        this->get_executor(), max_iters_, rel_residual_goal_, base));
-    cg->set_preconditioner(precond_factory_->generate(base));
-    return std::move(cg);
-}
-
-
 #define GKO_DECLARE_CG(_type) class Cg<_type>
-#define GKO_DECLARE_CG_FACTORY(_type) class CgFactory<_type>
+// #define GKO_DECLARE_CG_FACTORY(_type) class CgFactory<_type>
```

Comment:
These are commented out - why? should they be removed?

Reply:
Yes, removed.